### ISSUE TITLE
Re-enable helm3-library catalog, helm version is helm-v2

### ISSUE THREAD

**Issue by rootwuj** [rancher/rancher#27282]
**What kind of request is this (question/bug/enhancement/feature request):**
bug

**Steps to reproduce (least amount of steps as possible):**
1. Go to Global Catalogs, Tools -> Catalogs
2. Select the `helm3-library` catalog, click Edit, show `Helm Version` is `Helm v3`.
3. Delete `helm3-library` catalog, the catalog is not deleted, instead it goes into a Disabled status.
4. Select the `helm3-library` catalog, click Enable.

**Result:**
The `Helm Version` of `helm3-library` shows as `Helm v2`, it should be `Helm v3`.

<img width="1317" alt="image" src="https://user-images.githubusercontent.com/42061003/83028172-a776ab00-a063-11ea-8e5e-e7e2f664f545.png">

**Other details that may be helpful:**

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): `2.4 master-head (05/27/2020)`  d9184e914
- Installation option (single install/HA): single install

**Comment by jiaqiluo** [comment]
this bug is reproduced in both v2.4-head and master-head

Before disabling the `system-chart` catalog, it shows as a helm3 catalog and has the filed  `helmVersion": "helm_v3"`

<img width="204" alt="Screen Shot 2020-05-27 at 10 23 52 AM" src="https://user-images.githubusercontent.com/6218999/83052617-6dc28780-a004-11ea-8312-e4addbdfd397.png">

<img width="1401" alt="Screen Shot 2020-05-27 at 10 23 06 AM" src="https://user-images.githubusercontent.com/6218999/83052611-6c915a80-a004-11ea-8d3b-44ba2eee9915.png">

After disabling and re-enabling it, the field  `helmVersion": "helm_v3"` is gone, so helm_v2 is used by default.

**Comment by brendarearden** [comment]
I monitored the network activity and the payload when it is being re-enabled is this: 

```{type: "catalog", name: "helm3-library", url: "https://git.rancher.io/helm3-charts", branch: "master",…}
branch: "master"
kind: "helm"
name: "helm3-library"
type: "catalog"
url: "https://git.rancher.io/helm3-charts"

so it doesn't look like we are providing the helm version on the enable action - I believe this is something that is managed in the ui

**Comment by jiaqiluo** [comment]
The bug fix is validated in v2.4-head and master-head with the latest UI 

when re-enabling the helm3-library, the request payload looks like this 
```
branch: "master"
helmVersion: "helm_v3"
kind: "helm"
name: "helm3-library"
type: "catalog"
url: "https://git.rancher.io/helm3-charts"
```
and the catalog is re-enabled as helm3 catalog

![image](https://user-images.githubusercontent.com/6218999/83285710-8d3bea80-a193-11ea-9fa1-545f5f87d495.png)
